src/Propel/Generator/Command/Helper/ConsoleHelper.php

```diff
@@ -158,4 +160,19 @@ public function writeln($messages, $options = 0)
     {
         $this->output->writeln($messages, $options);
     }
+
+    /**
+     * @param string $question
+     * @param string|null $default
+     *
+     * @return string
+     */
+    private function formatQuestion($question, $default = null)
+    {
+        if ($default) {
+            return sprintf('<info>%s</info> [<comment>%s</comment>]: ', $question, $default);
+        } else {
```

Comment:
Not a blocker, but we don't need this else statement. We can just return a string without default after or even we can prepare it beforehand and add default part if default is not null.

Reply:
The else can indeed be removed, just return afterwards without else.

Reply:
TBH, I didn't do anything to this class but renaming it from `ConsoleHelper3` to `ConsoleHelper`.

Do you expect I improve it?

Reply:
Improving is always nice, but no requirement here.